test/controllers/api/v2/subscriptions_controller_test.rb

```diff
@@ -63,6 +64,14 @@ def test_index_protected
       end
     end
 
+    def test_index_with_activation_key_id
+      activation_key = ActivationKey.new
+      Katello::ActivationKey.stubs(:find_by).returns(activation_key)
+      get :index, activation_key_id: 1
+      assert_response :success
+      assert_template 'api/v2/subscriptions/index'
```

Comment:
Is there any way to test that only the subscriptions related to the activation key are being returned as opposed to all of them?